debezium-server-http/src/main/java/io/debezium/server/http/HttpChangeConsumer.java

```diff
@@ -142,6 +145,10 @@ void initWithConfig(Config config) throws URISyntaxException {
                 JWTAuthenticatorBuilder builder = JWTAuthenticatorBuilder.fromConfig(config, PROP_AUTHENTICATION_PREFIX);
                 authenticator = builder.build();
             }
+            else if (t.equalsIgnoreCase(STANDARD_WEBHOOKS_AUTHENTICATION)) {
```

Comment:
An optional one: I would extract a method with all the logic to get the `authenticator`.